bin.src/generateCoefficients.py

```diff
@@ -6,6 +6,7 @@
 from lsst.sims.movingObjects import Orbits
 from lsst.sims.movingObjects import ChebyFits
 
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description="Generate Chebyshev polynomial coefficients" +
```

Comment:
You can do that already :) It works slightly differently to how you hacked it into the old code, but if you specify a 30 day "tSpan" with a longer "tEnd", it runs from tStart to tEnd, in steps of tSpan.